jormungandr-lib/src/interfaces/certificate.rs

```diff
@@ -217,6 +224,11 @@ impl property::Serialize for SignedCertificate {
                 writer.write_all(c.serialize().as_slice())?;
                 writer.write_all(a.serialize_in(ByteBuilder::new()).finalize().as_slice())?;
             }
+            certificate::SignedCertificate::EvmMapping(c, a) => {
+                writer.write_all(&[15])?;
```

Comment:
why 15?

Reply:
From what I can tell, the serialization of these certificates use the same numbering scheme as the tag enum for fragments in chain-libs

https://github.com/input-output-hk/chain-libs/blob/master/chain-impl-mockchain/src/fragment/mod.rs#L53-L70

Reply:
Do they? most of `Certificate` and `SignedCertificate` overlap anyway and those are at two different levels